### ISSUE TITLE
smart quotes

### ISSUE THREAD

**Issue by mor1** [ocaml/omd#90]
similar to #84 please could something reasonable be done with ' and " a la latex

something like 

```
... "blah --> ... &ldquo;blah
...blah" ... --> ... blah&rdquo;...
... 'blah --> ... &lsquo;blah
...blah' ... --> ...blah&rsquo; ...
x'x -> x&rsquo;x
```

(unless there's a better html entity in the last case?)

(oh, and similarly handle the cases where punctuation is adjacent to the quote element too.)

**Comment by nojb** [comment]
This is not standard markdown; moreover it is something that seems like it can be handled outside the library without too much trouble, so closing. Please re-open if needed.

**Comment by mor1** [comment]
Well, I'm not really using OMD now anyway so the point is moot I guess :)
But what would be the option you'd recommend for handling outside the library?
Also, is there a canonical standard for markdown -- smartquotes is an option supported by, eg., pandoc and kramdown (one of the two recommended for use by jekyll for github pages) which I think are pretty widely used.

**Comment by nojb** [comment]
Indeed is is a common extension, and we may support it in the future (I added your suggestion to #205), but for the next release we are aiming to support the current Markdown standard, Commonmark: https://spec.commonmark.org/. Extensions are coming after that.

As for specific recommendations -- sorry I don't have anything specific in mind :( I amended my comment above to indicate that this is an extension that we may support in the future (after we ship 2.0). Thanks!
